app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,55 @@
+class MerchantsController < ApplicationController
+
+  def index
+    @merchants = Merchant.all
+  end
+
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def new
+
+  end
+
+  def create
+    @merchant = Merchant.create(merchant_params)
+    redirect_to "/merchants"
+  end
+
+  def edit
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def update
+    merchant = Merchant.find(params[:id])
+    merchant.update(merchant_params)
+    redirect_to "/merchants"
+  end
+
+  def delete
+    (Merchant.find(params[:id]).items).destroy_all
+    Merchant.destroy(params[:id])
+    redirect_to '/merchants'
+  end
+
+  def items
+    if params[:active] == "true"
```

Comment:
Looking at all three branches of this if statement, you do some form of `Merchant.find(params[:id])`. This indicates to me that you could put this line before the if statement to dry up the code.